Gemfile.lock

```diff
@@ -39,40 +45,6 @@ PATH
   remote: .
   specs:
     chef (18.0.142)
-      addressable
```

Comment:
This should probably not be getting removed, was `rake dependencies:update` used to update the lockfile?

Reply:
I did not specifically remove that. It came out during a bundle update?

Reply:
It's still in the file, oddly. The PR just looks funny. If you look down to the bottom of that section you'll see the library still listed. It also shows up when you view the full file.